pkg/util/webhooks/validating-webhooks/validating-webhook.go

```diff
@@ -93,3 +101,26 @@ func Serve(resp http.ResponseWriter, req *http.Request, admitter Admitter) {
 		return
 	}
 }
+
+func getContextFromRequest(req *http.Request) (context.Context, context.CancelFunc, error) {
+	ctx := req.Context()
+	if _, timeoutDefined := ctx.Deadline(); timeoutDefined {
+		return ctx, nil, nil
+	}
+
+	timeout := 10 * time.Second
+	if timeoutStr := req.URL.Query().Get("timeout"); timeoutStr != "" {
+		parsedTimeout, err := time.ParseDuration(timeoutStr)
+		if err != nil {
+			errTxt := fmt.Sprintf("failed to parse timeout duration of %q", timeoutStr)
+			log.Log.Reason(err).Error(errTxt)
```

Comment:
Should we log the error here? I'm not sure. What's the typical behavior? The calling function already tracks it by sending it to the client.

Reply:
This is some kind of security reasoning.

I want to log the original error, but not to send it to the user.